src/interfaces/userRequest.ts

```diff
@@ -1,108 +1,216 @@
 import { TypedDataDomain, TypedDataField } from 'ethers'
 import { SiweMessage as ViemSiweMessage } from 'viem/siwe'
 
-import { Session } from '../classes/session'
+import { AccountOp } from '../libs/accountOp/accountOp'
+import { SubmittedAccountOp } from '../libs/accountOp/submittedAccountOp'
 import { PaymasterService } from '../libs/erc7677/types'
 import { AccountId } from './account'
-import { SignMessageAction } from './actions'
 import { AutoLoginStatus, SiweValidityStatus } from './autoLogin'
 import { Dapp, DappProviderRequest } from './dapp'
 import { Hex } from './hex'
 import { EIP7702Signature } from './signatures'
 
-export interface Calls {
-  kind: 'calls'
-  calls: {
-    to: string
-    value: bigint
-    data: string
-    id?: string
+// @TODO: move this type and it's deps (PlainTextMessage, TypedMessage) to another place,
+// probably interfaces
+export interface Message {
+  fromRequestId: string | number
+  content:
+    | (PlainTextMessageUserRequest['meta']['params'] & {
+        kind: PlainTextMessageUserRequest['kind']
+      })
+    | (TypedMessageUserRequest['meta']['params'] & { kind: TypedMessageUserRequest['kind'] })
+    | (AuthorizationUserRequest['meta']['params'] & { kind: AuthorizationUserRequest['kind'] })
+    | (SiweMessageUserRequest['meta']['params'] & { kind: SiweMessageUserRequest['kind'] })
+
+  accountAddr:
+    | PlainTextMessageUserRequest['meta']['accountAddr']
+    | TypedMessageUserRequest['meta']['accountAddr']
+    | AuthorizationUserRequest['meta']['accountAddr']
+    | SiweMessageUserRequest['meta']['accountAddr']
+  chainId:
+    | PlainTextMessageUserRequest['meta']['chainId']
+    | TypedMessageUserRequest['meta']['chainId']
+    | AuthorizationUserRequest['meta']['chainId']
+    | SiweMessageUserRequest['meta']['chainId']
+  signature: EIP7702Signature | string | null
+}
```

Comment:
This type is very hard to read... Let's simplify accountAddr and chainId for now and think about content in the future:
```
  accountAddr: string
  chainId: bigint
```